machine_management/index-test.adoc

```diff
@@ -0,0 +1,43 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="index-test-gh-comments"]
+= Test file to mess with for DOL
+include::_attributes/common-attributes.adoc[]
+:context: index-test-gh-comments
+
+toc::[]
+
+Here is a file to mess with to try to accept comments directly via GH review features.
+Text from the link:https://www.lipsum.com[Lorem Ipsum] generator.
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean consectetur placerat purus vel pulvinar. Ut id dignissim erat. Nam volutpat consequat posuere. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce massa dolor, interdum egestas imperdiet non, accumsan id lorem. Sed sed ultrices quam, nec aliquet orci. Cras sodales ac quam id hendrerit. Vivamus et eleifend ipsum, sed iaculis lorem. Nulla scelerisque, ex a dapibus dignissim, ipsum libero hendrerit neque, ac scelerisque odio sapien quis ipsum. Maecenas in hendrerit risus. Nullam eget sapien massa. Donec magna quam, vehicula eu elementum at, volutpat et est. Cras vel suscipit nisi, aliquam congue mauris. In fermentum nulla sed tortor imperdiet tristique.
+
+[NOTE]
+====
+Nam nec dolor ut orci ultrices volutpat eu sit amet urna. In diam quam, aliquam ut tristique id, laoreet id tortor. Aliquam convallis ac lorem varius elementum. Integer auctor velit pretium, molestie turpis et, ornare arcu. Nam non orci at enim fringilla congue eget nec tortor. Duis tempor finibus felis quis bibendum. Vestibulum a metus aliquam, sollicitudin felis id, sagittis ante.
```

Comment:
```suggestion
The Wabanaki Confederacy (Waponahki) — translated as “People of the First Light” or “Dawnland” — currently comprises five principal nations: the Mi’kmaq, Maliseet, Passamaquoddy, Penobscot, and Abenaki, and stretches from Newfoundland in the north, to mid-Maine in the south, and parts of Quebec in the west. Historically, the confederacy united five Algonquin language-speaking Indian tribes. While the Wabanaki Confederacy was disbanded in 1862, the Mi’kmaq, Maliseet, Passamaquoddy, Penobscot nations still exist and remain closely aligned, in part because all peoples claiming Wabanaki lineage have forbearers from multiple Wabanaki and colonial ancestries.
```

Reply:
And a lot of French and a few Scottish people mixed all up in there.